### DISCUSSION TITLE
Unable to switch search input method

### DISCUSSION THREAD

**Discussion by soutou233** [shedaniel/RoughlyEnoughItems#1259] [Q & A]
Plugin Version: RoughlyEnoughItems-8.3.576
Game Version: 1.18.2
ModPack: ATM7 to the sky

As shown in the image, there has no option for switch input language(PinYin)
![image](https://user-images.githubusercontent.com/38411083/205845853-a10f9630-47f0-4f6e-a285-55f2ea388409.png)

**Comment by shedaniel** [comment]
Can you share your logs?

**Reply by Dr-WeiAL** [reply]
I have the same problem
[rei.log](https://github.com/shedaniel/RoughlyEnoughItems/files/10201938/rei.log)
